### DISCUSSION TITLE
Setting carousel indicator color

### DISCUSSION THREAD

**Discussion by gplhegde** [themesberg/flowbite#104] [Q&A]
I am trying to get the carousel working. I see from the [source code](https://github.com/themesberg/flowbite/blob/main/src/components/carousel.js#L194) that the background color of the active slide indicator is hard coded to `bg-white` and that of the inactive slide indicators are set to `bg-white/50`. Is there any way to specify the active and inactive colors to some other classes? The indicators are not visible on white containers.

**Comment by yuxufm** [comment]
I edited the code like this and it's working for changing active and inactive indicators colors

      this._indicators.map(function (indicator) {
        indicator.el.setAttribute('aria-current', 'false');
        indicator.el.classList.remove('bg-[#BB9F65]', 'bg-[#BB9F65]');
        indicator.el.classList.add('bg-[#BB9F65]/50', 'bg-[#BB9F65]/50');
    
        if (indicator.id === id) {
          indicator.el.classList.add('bg-[#BB9F65]', 'bg-[#BB9F65]');
          indicator.el.classList.remove('bg-[#BB9F65]/50', 'bg-[#BB9F65]/50');
          indicator.el.setAttribute('aria-current', 'true');
        }
